src/main/java/org/openstreetmap/atlas/geography/Polygon.java

```diff
@@ -162,7 +160,9 @@ public Location center()
      */
     public boolean fullyGeometricallyEncloses(final Location location)
     {
-        return awtPolygon().contains(location.asAwtPoint());
+        final com.vividsolutions.jts.geom.Polygon polygon = new JtsPolygonConverter().convert(this);
```

Comment:
It looks like we have a static converter already initialized above. We can use it instead of creating a new one.